domains/pallets/evm-tracker/src/weights.rs

```diff
@@ -0,0 +1,51 @@
+
+//! Autogenerated weights for `pallet_evm_tracker`
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 46.2.0
+//! DATE: 2025-05-28, STEPS: `50`, REPEAT: `20`, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! WORST CASE MAP SIZE: `1000000`
+//! HOSTNAME: `Genes-MacBook-Pro.local`, CPU: `<UNKNOWN>`
```

Comment:
I should replace this hostname with something more descriptive, and maybe the CPU as well.